kubernetes/linux/Dockerfile

```diff
@@ -18,5 +19,11 @@ ENV KUBE_CLIENT_BACKOFF_DURATION 0
 RUN /usr/bin/apt-get update && /usr/bin/apt-get install -y libc-bin wget openssl curl sudo python-ctypes init-system-helpers  net-tools rsyslog cron vim dmidecode apt-transport-https gnupg && rm -rf /var/lib/apt/lists/*
 COPY setup.sh main.sh defaultpromenvvariables defaultpromenvvariables-rs $tmpdir/
 WORKDIR ${tmpdir}
+
+# copy docker provider shell bundle to use the agent image
+COPY ./Linux_ULINUX_1.0_x64_64_Release/docker-cimprov-10.0.0-*.x86_64.sh .

```

Comment:
hard coded to 10.0.0.* ? this could change to 11.0.0.5 for example....

Reply:
> hard coded to 10.0.0.* ? this could change to 11.0.0.5 for example....

> removed hard coded dependency and updated make file to purge any old shell bundle.